ntbs-service/Program.cs

```diff
@@ -63,20 +63,23 @@ private static void SetUpLogger()
                 .CreateLogger();
         }
 
-        private static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .ConfigureKestrel(options =>
-                {
-                    options.AddServerHeader = false;
-                })
-                .UseStartup<Startup>()
-                .UseSentry(options =>
-                {
-                    options.Release = Environment.GetEnvironmentVariable(Constants.Release);
-                    // This is a workaround for a known issue in Sentry.
-                    // See https://github.com/getsentry/sentry-dotnet/issues/1210
-                    options.DisableDiagnosticSourceIntegration();
-                })
+        private static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHost(webBuilder => webBuilder
+                    .UseKestrel()
+                    .ConfigureKestrel(options =>
+                    {
+                        options.AddServerHeader = false;
+                    })
+                    .UseStartup<Startup>()
+                    .UseSentry(options =>
+                    {
+                        options.Release = Environment.GetEnvironmentVariable(Constants.Release);
+                        // This is a workaround for a known issue in Sentry.
+                        // See https://github.com/getsentry/sentry-dotnet/issues/1210
+                        options.DisableDiagnosticSourceIntegration();
+                    })
+                )
```

Comment:
This is the main change which now allows us to put `UseSerilog()` in the right place